velox/vector/arrow/tests/ArrowBridgeArrayTest.cpp

```diff
@@ -1472,15 +1472,18 @@ class ArrowBridgeArrayImportTest : public ArrowBridgeArrayExportTest {
   }
 
   template <typename F>
-  void testArrowRoundTrip(const arrow::Array& array, F validateVector) {
+  void testArrowRoundTrip(
+      const arrow::Array& array,
+      F validateVector,
+      const ArrowOptions& options = ArrowOptions{}) {
```

Comment:
this is probably a better pattern. Can you remove the "options_" member then?

Reply:
Done